load/loader.go

```diff
@@ -138,7 +142,41 @@ func (l *CommonBenchmarkRunner) postRun(wg *sync.WaitGroup, start *time.Time) {
 	// Wait for all workers to finish
 	wg.Wait()
 	end := time.Now()
-	l.summary(end.Sub(*start))
+	took := end.Sub(*start)
+	l.summary(took)
+	if len(l.BenchmarkRunnerConfig.ResultsFile) > 0 {
```

Comment:
```suggestion
	if l.BenchmarkRunnerConfig.ResultsFile != "" {
```
Check against default value.